subquery.py

```diff
@@ -94,7 +95,20 @@ def query_multiple_source(
     if discourse:
         raise NotImplementedError
     if gdrive:
-        raise NotImplementedError
+        gdrive_query_engine = GDriveQueryEngine(community_id=community_id).prepare()
+        tool_metadata = ToolMetadata(
+            name="Google-Drive",
+            description=(
+                "Stores and manages documents, spreadsheets, presentations,"
```

Comment:
The description should be for the query engine, not for google drive itself. So I'd suggest a slightly different wording:

"Contains contents of documents, spreadsheets and presentations of the community"

Also, do we access text from the spreadsheets and presentations too or only from the documents?

Reply:
Within google-drive folder there can be spreadsheets, presentations and docx files. So, when reading them in ETL job, we can read them.

Reply:
What does the output for a presentation or spreadsheet look like? How do we chunk the contents? If the spreadsheet contains a bunch of numbers, do we embed them?